openshift-template.yaml

```diff
@@ -257,7 +257,10 @@ objects:
     spec:
       host: ${FRONTEND_HOSTNAME}
       port:
-        targetPort: web
+        targetPort: http
```

Comment:
Doesn't the port name here have to match the port name given in the service definition? That is, "web".

Reply:
That's right. I'll fix that.